inc/user.class.php

```diff
@@ -4049,7 +4049,7 @@ static function dropdown($options = []) {
 
       $view_users = self::canView();
 
-      if ($p['value'] == 'myself') {
+      if (strval($p['value']) == 'myself') {
```

Comment:
Should not this be solved like:
```suggestion
      if ($p['value'] === 'myself') {
```